addons/payment_demo/const.py

```diff
@@ -1,6 +1,6 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 # The codes of the payment methods to activate when Demo is activated.
-DEFAULT_PAYMENT_METHOD_CODES = [
+DEFAULT_PAYMENT_METHODS_CODES = [
```

Comment:
though, the difference is not so big in terms of performance as  long as the list is short which is the case. But I agree that for our purposes set is more appropriate. Thanks

Reply:
Not big at all for these kinds of things, and had that be the only comment I had to make, I wouldn't have. It's almost just a personal preference, but since it's something I try to use more often recently, I thought you would be interested too ^^